pkg/storaged/pages.jsx

```diff
@@ -591,7 +592,7 @@ export const PageTable = ({ emptyCaption, aria_label, pages, crossrefs, sorted,
                     <CardBody>
                         <Split hasGutter>
                             { icon && <SplitItem>{icon}</SplitItem> }
-                            <SplitItem isFilled><strong>{name}</strong>{info}</SplitItem>
+                            <SplitItem isFilled><strong>{name.length > 40 ? <Truncate content={name} trailingNumChars={6} position='middle' /> : name}</strong>{info}</SplitItem>
```

Comment:
Let's use `Truncate` unconditionally and not just for names longer than 40 characters.

Reply:
So, I don't super like the tooltip, tbh. People can see the full name in the details page, and truncating in the middle plus the device name will hopefully be enough to identify a row enough.

We could use the HTML variant of truncate, which would allow us to get rid of the tooltip.

What do you think?